packit_service/models.py

```diff
@@ -149,13 +150,78 @@ def get_or_create(
                 session.add(project)
             return project
 
+    @classmethod
+    def get_projects_range(cls, first: int, last: int):
+        with get_sa_session() as session:
+            projects = session.query(GitProjectModel).order_by(
+                GitProjectModel.namespace
+            )[first:last]
+            return projects
+
+    @classmethod
+    def get_project_prs_range(
+        cls, first: int, last: int, namespace: str, repo_name: str
```

Comment:
Relying on `namespace` and `repo_name` is going to be problematic as we start running Packit Service on more and more forges, as we might have projects stored in the same namespace and a repo called the same way but on different forges.

I see to ways to improve this:
- use `git_projects.id` here directly OR
- introduce a `forge` argument—but `GitProjectModel` is missing this field, so for this to work, that should be introduced first.

Reply:
Youre right, namespace/repo_name isnt unique enough. I didnt think of other forges.

> introduce a forge argument—but GitProjectModel is missing this field, so for this to work, that should be introduced first.

Ideal. 
But this will involve labelling all previous data as well (or maybe default to github) and might end up being too much work.

> use git_projects.id here directly OR

Quick and easy but your point in the first message stands. Prolly needs to be discussed.